apis/core/hydra/dimension-metadata.ttl

```diff
@@ -0,0 +1,43 @@
+BASE           <urn:hydra-box:api>
+PREFIX cc:     <https://cube-creator.zazuko.com/vocab#>
+prefix hydra:  <http://www.w3.org/ns/hydra/core#>
+prefix sh:     <http://www.w3.org/ns/shacl#>
+prefix schema: <http://schema.org/>
+prefix code:   <https://code.described.at/>
+
+cc:DimensionMetadataCollection
+  a
+    hydra:Class ;
+  hydra:supportedProperty
+    [ hydra:property
+        schema:hasPart ] ;
+  hydra:supportedOperation
+    _:DimensionUpdateOperation ;
+.
+
+schema:hasPart
+  hydra:supportedOperation
+    _:DimensionUpdateOperation .
```

Comment:
It's slightly weird how the operation is attached to both the collection as well as the `schema:hasPart` property.

Only the latter should be necessary (request to the objects of this property) but hydra-box does not recognise URIs with query strings `?dimension=XYZ` as an independent resource and instead looks for operations on an ID with the query parameters removed.

cc @bergos

Reply:
Back with my question of the other day then: do we really need these URIs with query string? What do they bring us over URIs with a random value for example?

Reply:
Yes, you do make a compelling case. There is nothing in favour of `?dimension=foo` vs any URI constructed without a query string